src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  width: auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-sizing: border-box;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  margin-left: 50px;
+}
+
+.nav {
+  margin-right: 50px;
+  display: flex;
```

Comment:
It's necessary? What will change if you remove all flex styles for the nav class?